back/engines/commercial/texting/.gitignore

```diff
@@ -0,0 +1,11 @@
+/.bundle/
```

Comment:
It's definitely not necessary, but
1. It's created by default and we have it in other engines
2. It will be useful if somebody will use rails commands (`dummy` app) inside of the engine. So, `/test/dummy/tmp` won't be added to the repo.